frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView.stories.tsx

```diff
@@ -94,6 +97,7 @@ const CARD_TABLE_ID = getNextId();
 const TAB_ID = getNextId();
 const PARAMETER_ID = "param-hex";
 const initialState = createMockState({
+  currentUser: null,
```

Comment:
This is needed otherwise object detail would not render.
https://github.com/metabase/metabase/blob/a0f527b20b6f7a4c1827bf854cde0635bfebb1b0/frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetail.tsx#L30-L34